lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -1269,6 +1269,95 @@ void replaceCallOps(handshake::FuncOp f, ConversionPatternRewriter &rewriter) {
   }
 }
 
+/// Rewrite affine.for in a handshake.func into its standard form. Affine
+/// analysis should be carried out before this function being called.
+LogicalResult rewriteAffineFor(handshake::FuncOp f,
+                               ConversionPatternRewriter &rewriter) {
+  // Get all affine.for operations in the function body.
+  SmallVector<Operation *, 8> forOps;
```

Comment:
Please document what 'standard form' is.  I think you mean "it's representation as a CFG in the standard dialect"?
How are you planning to pass the affine dependence information to the point where the standard dialect is further lowered into the handshake dialect?

Reply:
> Please document what 'standard form' is. I think you mean "it's representation as a CFG in the standard dialect"?

Yes that is correct. Sorry for abusing these terms around.

> How are you planning to pass the affine dependence information to the point where the standard dialect is further lowered into the handshake dialect?

Thanks for asking. I will do affine analysis before `rewriteAffineFor` being called and store all the dependence information.